### ISSUE TITLE
DM installation issues 

### ISSUE THREAD

**Issue by SofidelAustin** [SofidelAustin/Display-Monkey-webconfig-file#1]
This is what my DM management file looks like:
"<add connectionString="metadata=res://*/Models.DMModel.csdl|res://*/Models.DMModel.ssdl|res://*/Models.DMModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=.\MSSQLSERVER;initial catalog=DisplayMonkey;persist security info=True;user id=DisplayMonkey;password=Password!;MultipleActiveResultSets=True;App=EntityFramework&quot;" name="DisplayMonkeyEntities" providerName="System.Data.EntityClient" />
        <add connectionString="data source=SIGNAGE\MSSQLSERVER;Integrated Security=SSPI;AttachDBFilename=|DataDirectory|aspnetdb.mdf;User Instance=true" name="LocalSqlServer" providerName="System.Data.SqlClient" />
  </connectionStrings>"
I have only edited this portion the full web.config file is below. 

<?xml version="1.0" encoding="utf-8"?>

<configuration>
  <configSections>
    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
    
  </configSections>
  <connectionStrings>
        <remove name="LocalSqlServer" />
    <add connectionString="metadata=res://*/Models.DMModel.csdl|res://*/Models.DMModel.ssdl|res://*/Models.DMModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=.\MSSQLSERVER;initial catalog=DisplayMonkey;persist security info=True;user id=DisplayMonkey;password=Password!;MultipleActiveResultSets=True;App=EntityFramework&quot;" name="DisplayMonkeyEntities" providerName="System.Data.EntityClient" />
        <add connectionString="data source=SIGNAGE\MSSQLSERVER;Integrated Security=SSPI;AttachDBFilename=|DataDirectory|aspnetdb.mdf;User Instance=true" name="LocalSqlServer" providerName="System.Data.SqlClient" />
  </connectionStrings>
  <appSettings>
    <add key="webpages:Version" value="3.0.0.0" />
    <add key="webpages:Enabled" value="false" />
    <add key="PreserveLoginUrl" value="true" />
    <add key="ClientValidationEnabled" value="true" />
    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
    <add key="presentationSite" value="" />
    <add key="presentationRoot" value="DisplayMonkey" />
  </appSettings>
  
  <system.web>
    <compilation targetFramework="4.5.1">
      <assemblies>
        <add assembly="System.Data.Entity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
      </assemblies>
    </compilation>
    <httpRuntime targetFramework="4.5" maxRequestLength="1048576" executionTimeout="120000" />
    <authentication mode="Windows" />
    <authorization>
      <deny users="?" />
    </authorization>
    <pages>
      <namespaces>
        <add namespace="System.Web.Helpers" />
        <add namespace="System.Web.Mvc" />
        <add namespace="System.Web.Mvc.Ajax" />
        <add namespace="System.Web.Mvc.Html" />
        <add namespace="System.Web.Optimization" />
        <add namespace="System.Web.Routing" />
        <add namespace="System.Web.WebPages" />
      </namespaces>
    </pages>
    <customErrors mode="On" />
    <globalization enableClientBasedCulture="true" uiCulture="auto" culture="auto" />
  </system.web>
  <system.webServer>
    <security>
      <requestFiltering>
        <requestLimits maxAllowedContentLength="1073741824" />
      </requestFiltering>
    </security>
    <validation validateIntegratedModeConfiguration="false" />
    <modules runAllManagedModulesForAllRequests="true" />
    <httpErrors errorMode="Detailed" />
    <httpProtocol>
      <customHeaders>
        <add name="X-UA-Compatible" value="IE=11" />
      </customHeaders>
    </httpProtocol>
    <handlers>
      <remove name="ExtensionlessUrlHandler-Integrated-4.0" />
      <remove name="OPTIONSVerbHandler" />
      <remove name="TRACEVerbHandler" />
      <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />
    </handlers>
  </system.webServer>
  <runtime>
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <dependentAssembly>
        <assemblyIdentity name="WebGrease" publicKeyToken="31bf3856ad364e35" />
        <bindingRedirect oldVersion="0.0.0.0-1.6.5135.21930" newVersion="1.6.5135.21930" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="Antlr3.Runtime" publicKeyToken="eb42632606e9261f" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-3.5.0.2" newVersion="3.5.0.2" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.Helpers" publicKeyToken="31bf3856ad364e35" />
        <bindingRedirect oldVersion="1.0.0.0-3.0.0.0" newVersion="3.0.0.0" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.WebPages" publicKeyToken="31bf3856ad364e35" />
        <bindingRedirect oldVersion="1.0.0.0-3.0.0.0" newVersion="3.0.0.0" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.Mvc" publicKeyToken="31bf3856ad364e35" />
        <bindingRedirect oldVersion="1.0.0.0-5.2.7.0" newVersion="5.2.7.0" />
      </dependentAssembly>
    </assemblyBinding>
  </runtime>
  <entityFramework>
    <defaultConnectionFactory type="System.Data.Entity.Infrastructure.LocalDbConnectionFactory, EntityFramework">
      <parameters>
        <parameter value="v11.0" />
      </parameters>
    </defaultConnectionFactory>
    <providers>
      <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
    </providers>
  </entityFramework>
</configuration>

**Comment by michaelmalmgren** [comment]
Austin, I cannot see a web.config or the content of one? Also what and where to you get an error?

**Comment by SofidelAustin** [comment]
[New Text Document.txt](https://github.com/SofidelAustin/Display-Monkey-webconfig-file/files/4165228/New.Text.Document.txt)

**Comment by SofidelAustin** [comment]
> 
> 
> Austin, I cannot see a web.config or the content of one? Also what and where to you get an error?

Are you able to see the document I attached in the comment on this thread?
Thanks.

**Comment by michaelmalmgren** [comment]
Yes, I do see the attachment. Now if you can elaborate on the error you're getting I'll try to help you.
